ipa-core/src/protocol/ipa_prf/mod.rs

```diff
@@ -867,7 +867,7 @@ mod compact_gate_tests {
     fn step_count_limit() {
         // This is an arbitrary limit intended to catch changes that unintentionally
         // blow up the step count. It can be increased, within reason.
-        const STEP_COUNT_LIMIT: u32 = 24_000;
+        const STEP_COUNT_LIMIT: u32 = 36_000;
```

Comment:
This was failing - not sure if this is an increase "within reason". If I understand this test correctly, this went up because we added a whole new protocol with a bunch of steps, so it makes sense we'd need to increase this.

cc @akoshelev @andyleiserson

Reply:
lets set it down to 33k